exercises/015-maps/maps.go

```diff
@@ -29,10 +29,10 @@ func main() {
 
 	fmt.Println(age)
 
-	// When accessing a map be carefult with
-	// the key- it must be an exact match to the
-	// key you are searching.
+	// When accessing a map be careful with
+	// the key- it must be an exact match of the
+	// key you are searching for.
 	// If the key is not found then the access
-	// yields the zero value for the element type.
+	// yields a zero value for the element type.
```

Comment:
Good changes all around! Little nitpick though: in Go we refer to "the zero value of the type" since there is but one zero value for a given type.

Reply:
The map returns the zero value of the element type if the key is not in the map.

The zero value depends on the element type of the map. In the case above the map returns 0 because the elements are ints and 0 is the zero value for ints and floats. Other types have different zero values, for example the zero value for a string is the empty string, zero val for a pointer is nil.